pyproject.toml

```
@@ -25,8 +26,9 @@ classifiers = [
   "Typing :: Typed",
 ]
 requires-python = ">= 3.11, < 4"
+# TODO(cookiecutter): Remove and add more dependencies if appropriate
 dependencies = [
-  "frequenz-api-common >= 0.5.0, < 0.6.0",
+  "frequenz-api-common >= 0.3.1, < 0.4.0",
```

You probably don't want to downgrade this.
```suggestion
  "frequenz-api-common >= 0.5.0, < 0.6.0",
```